### ISSUE TITLE
Remove a docstring from members of namedtuple

### ISSUE THREAD

**Issue by tk0miya** [sphinx-doc/sphinx#7353]
**Is your feature request related to a problem? Please describe.**
All members of namedtuple have a similar docstring: "Alias for field number N".

```
$ python
Python 3.8.1 (default, Feb 13 2020, 13:34:23)
[Clang 11.0.0 (clang-1100.0.33.17)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from collections import namedtuple
>>>
>>> Coordinate = namedtuple('Coordinate', 'X Y')
>>> Coordinate.__doc__
'Coordinate(X, Y)'
>>> Coordinate.X.__doc__
'Alias for field number 0'
>>> Coordinate.Y.__doc__
'Alias for field number 1'
```

This would be helpful for online document tool like `help()`. But it is a bit redundant for the built document.
![https___qiita-image-store s3 ap-northeast-1 amazonaws com_0_57661_68445eb3-ff98-dbb9-d844-e5f2aac31139](https://user-images.githubusercontent.com/748828/77231422-266fe300-6bde-11ea-9f68-182a8d74cb11.png)

**Describe the solution you'd like**
Remove a docstring from members of namedtuple.

**Describe alternatives you've considered**
N/A

**Additional context**
refs: https://chrisdown.name/2015/09/20/removing-namedtuple-docstrings-from-sphinx.html

- [e.g. URL or Ticket]

**Comment by shimizukawa** [comment]
Agreed. I also don't think documentation of namedtuple attributes is necessary.
